src/Stats.h

```diff
@@ -1,14 +1,15 @@
 #pragma once
 
 #include <QLineEdit>
-#define NUMBER_OF_INT_STATS 2
-#define NUMBER_OF_FLOAT_STATS 1
-#define NUMBER_OF_STATS 4
+#define NUMBER_OF_INT_STATS 6
+#define NUMBER_OF_FLOAT_STATS 3
+#define NUMBER_OF_STATS 10
+#define UNSOLVED_SCORE 1000
 
 namespace mms {
 
-enum StatsEnum{TOTAL_DISTANCE, TOTAL_TURNS, // integers
-               EFFECTIVE_DISTANCE, // floats
+enum StatsEnum{TOTAL_DISTANCE, TOTAL_TURNS, BEST_RUN_DISTANCE, BEST_RUN_TURNS, CURRENT_RUN_DISTANCE, CURRENT_RUN_TURNS, // integers
+               TOTAL_EFFECTIVE_DISTANCE, BEST_RUN_EFFECTIVE_DISTANCE, CURRENT_RUN_EFFECTIVE_DISTANCE, // floats
                SCORE // has a text box but is not saved in an array
```

Comment:
Nitpick: since this enum is getting quite large, you should put each value on its own line.

Also, consider using `enum class` since we should never actually need the integer values.